docs/zh/maintain/diagnose.md

```diff
@@ -81,12 +81,13 @@ openmldb_tool status --conn
 
 ```
 openmldb_tool inspect -h
-usage: openmldb_tool inspect [-h] [--helpfull] {online,offline} ...
+usage: openmldb_tool inspect [-h] [--helpfull] {online,offline,job} ...
 
 positional arguments:
-  {online,offline}
-    online          only inspect online table
-    offline         only inspect offline jobs, check the job log
+  {online,offline,job}
+    online              only inspect online table.
+    offline             only inspect offline jobs, check the job log
+    job                 show jobs by state or show joblog by id
```

Comment:
Already have the simple parse logic now, so it can be 'show joblog or parse joblog by id'